pom.xml

```diff
@@ -5,12 +5,13 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.12</version>
+    <version>3.42</version>
     <relativePath />
   </parent>
+
   <groupId>org.jenkins-ci.plugins</groupId>
   <artifactId>docker-commons</artifactId>
-  <version>1.15-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
```

Comment:
Ah whoops, I should have reincrementalified after the last release because the parent pom is older than the version where that start happening automatically (3.18), thanks for fixing it!

Reply:
Your welcome, but you should really thank the enforcer plugin for nagging about it